src/Traveler.js

```diff
@@ -0,0 +1,10 @@
+class Traveler {
```

Comment:
I wonder if some of your Trip methods could be moved to your Traveler class - if that would make it more understandable? Just a thought!